thoth/storages/graph/models.py

```diff
@@ -242,6 +242,7 @@ class PackageExtractRun(Base, BaseExtension):
     )
 
     found_python_files = relationship("FoundPythonFile", back_populates="package_extract_run")
+    found_python_interpreter = relationship("FoundPythonInterpreter", back_populates="package_extract_run")
```

Comment:
I've just spotted this - we try to respect plural in many-to-many relations, so please adjust to `found_python_interpreters` (here and bellow).